packages/indy-vdr/src/anoncreds/IndyVdrAnonCredsRegistry.ts

```diff
@@ -390,31 +396,196 @@ export class IndyVdrAnonCredsRegistry implements AnonCredsRegistry {
+      const indySdkPoolService = agentContext.dependencyManager.resolve(IndyVdrPoolService)
+      const did = didFromRevocationRegistryDefinitionId(revocationRegistryId)
 
-  public async getRevocationRegistryDefinition(
-    agentContext: AgentContext,
-    revocationRegistryDefinitionId: string
-  ): Promise<GetRevocationRegistryDefinitionReturn> {
-    return {
-      resolutionMetadata: {
-        error: 'Not Implemented',
-        message: `Revocation registry definition not yet implemented`,
-      },
-      revocationRegistryDefinitionId,
-      revocationRegistryDefinitionMetadata: {},
+      const pool = await indySdkPoolService.getPoolForDid(agentContext, did)
+
+      agentContext.config.logger.debug(
+        `Using ledger '${pool.indyNamespace}' to retrieve revocation registry deltas with revocation registry definition id '${revocationRegistryId}' until ${timestamp}`
+      )
+
+      const request = new GetRevocationRegistryDeltaRequest({
+        submitterDid: did,
+        revocationRegistryId,
+        toTs: timestamp,
+      })
+
+      agentContext.config.logger.trace(
+        `Submitting get revocation registry delta request for revocation registry '${revocationRegistryId}' to ledger`
+      )
+
+      const response = await pool.submitReadRequest(request)
+
+      agentContext.config.logger.debug(
+        `Got revocation registry deltas '${revocationRegistryId}' until timestamp ${timestamp} from ledger`
+      )
+
+      const { revocationRegistryDefinition, resolutionMetadata, revocationRegistryDefinitionMetadata } =
+        await this.getRevocationRegistryDefinition(agentContext, revocationRegistryId)
+
+      if (
+        !revocationRegistryDefinition ||
+        !revocationRegistryDefinitionMetadata.issuanceType ||
+        typeof revocationRegistryDefinitionMetadata.issuanceType !== 'string'
+      ) {
+        return {
+          resolutionMetadata: {
+            error: `error resolving revocation registry definition with id ${revocationRegistryId}: ${resolutionMetadata.error} ${resolutionMetadata.message}`,
+          },
+          revocationStatusListMetadata: {
+            didIndyNamespace: pool.indyNamespace,
+          },
+        }
+      }
+
+      const isIssuanceByDefault = revocationRegistryDefinitionMetadata.issuanceType === 'ISSUANCE_BY_DEFAULT'
+
+      if (!response.result.data) {
+        return {
+          resolutionMetadata: {
+            error: 'notFound',
+            message: `Error retrieving revocation registry delta '${revocationRegistryId}' from ledger, potentially revocation interval ends before revocation registry creation`,
+          },
+          revocationStatusListMetadata: {},
+        }
+      }
+
+      const revocationRegistryDelta = {
+        value: {
+          accum: response.result.data?.value.accum_to.value.accum,
+          issued: response.result.data?.value.issued,
+          revoked: response.result.data?.value.revoked,
+        },
+        ver: '1.0',
+      }
```

Comment:
Althoug this is fine, and it works, it would feel  a bit cleaner to me to only make the interface contain values that would actually be needed by the `anonCredsRevocationStatusListFromIndyVdr` method. 

E.g. I think the following value would work just as well:

```suggestion
      const revocationRegistryDelta = {
        accum: response.result.data?.value.accum_to.value.accum,
        issued: response.result.data?.value.issued,
        revoked: response.result.data?.value.revoked
      }
```